scripts/build_test.xml

```diff
@@ -101,16 +97,22 @@
 	</target>
 
 	<target name="stage_test_material" depends="-create_test_directory">
-		<condition property="copy-encoding" value="IBM-1047" else="UTF-8">
-			<isset property="isZOS"/>
-		</condition>
-		<copy todir="${ascii.test.dir}" outputencoding="${copy-encoding}">
-			<fileset dir="${ascii.workspace.dir}">
-				<include name="TKG/**" />
-				<include name="*.mk" />
-			</fileset>
-		</copy>
-
+		<if>
+			<equals arg1="${os.name}" arg2="z/OS"/>
+			<then>
+				<exec executable="cp">
+					<arg line="-rf ${ascii.workspace.dir} ${ascii.test.dir}" />
+				</exec>
```

Comment:
Please use multiple `<arg value="..." />` elements so argument separation does not depend on whether they contain spaces. I think we also want `failonerror`.
```
	<exec executable="cp" failonerror="true">
		<arg value="-rf" />
		<arg value="${ascii.workspace.dir}" />
		<arg value="${ascii.test.dir}" />
	</exec>
```

Please add a space before `/>` like most of this file.